scripts/third_party_size_check.py

```diff
@@ -74,4 +83,4 @@ def _check_third_party_size():
     print 'Running third-party size check'
     _check_third_party_size()
     print 'Third-party folder size check passed.'
-    print ''
+    print ''
```

Comment:
Why this change? You removed a newline. Again, please do not bypass pre-submit checks. It's not recommended.

Reply:
Sorry, I've reverted the changes